File: Estrutura_Condicional/C/Ex1042/main.c
#include <stdio.h>

#define max(a, b, c) (((a) > (b)) ? (((a) > (c)) ? (a) : (c)) : (((b) > (c)) ? (b) : (c)))
#define min(a, b, c) (((a) < (b)) ? (((a) < (c)) ? (a) : (c)) : (((b) < (c)) ? (b) : (c)))

int main(){

    int a, b, c;
    scanf("%d" "%d" "%d", &a, &b, &c);

    int menor = min(a, b, c);
    int meio = (a + b + c) - menor - max(a, b, c);
    int maior = max(a, b, c);

    printf("%d\n%d\n%d\n\n", menor, meio, maior);
    printf("%d\n%d\n%d\n", a, b, c);

    return 0;
}

File: Estrutura_Condicional/Java/Ex1042/Main.java
package Ex1042;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a, b ,c;
        a = sc.nextInt();
        b = sc.nextInt();
        c = sc.nextInt();
        sc.close();

        int menor = Math.min(a, b);
        int menorF = Math.min(menor, c);
        int maior = Math.max(a, b);
        int maiorF = Math.max(maior, c);
        int meio = (a + b + c) - menorF - maiorF;

        System.out.printf("%d\n%d\n%d\n", menorF, meio, maiorF);
        System.out.println();
        System.out.printf("%d\n%d\n%d\n", a, b, c);
    }
}